i18n/messages.json

```
@@ -399,6 +399,7 @@
   "documentCreationBehavior.error.invalidSubtype": "You cannot create this type of document in this location.",
   "documentCreationBehavior.error.noPermission": "You do not have permissions to create under this location.",
   "documentCreationBehavior.error.noSubtypes": "Select a different container to create your content.",
+  "documentCreationBehavior.error.trashedParent": "You cannot create this type of document under a trashed document.",
```

We cannot create any type of document on a trashed folder, right? Perhaps something like this would me shorter and more appropriate?:
```suggestion
  "documentCreationBehavior.error.trashedParent": "You cannot create content under a trashed document.",
```